src/main/java/com/arcbees/website/client/application/services/ServicesView.ui.xml

```diff
@@ -167,46 +167,88 @@
                 <h2>
                     <ui:msg description="Professional Services - Trusted By">Trusted By</ui:msg>
                 </h2>
-                <slider:Slider>
-                    <slider:content>
-                        <g:HTMLPanel>
-                            <ul>
-                                <li>
+                <div class="{res.style.mobileHidden}">
```

Comment:
@LarryMatte All you change between Mobile and Desktop slider is that one is using `<ul><li>` and the other one `<div>` ... Could you keep the desktop one and change `<li>` to `display: block` instead with media-queries once on mobile version? That way, you wouldn't need to duplicate the slider and the data in it

I also think that the grid should have general classes for "mobile hidden" "mobile only" and all, as we will eventually define the breakpoints in the grid settings. But that's another story.